README.md

```diff
@@ -8,23 +8,25 @@ It calculates how many pages it needs/can display for user to select from based
 
 ## Usage
 
-```
+```bash
 npm install ngx-paginate --save
 ```
 
 Import module
-```
+
+```bash
```

Comment:
In fact, this is also a Typescript snippet, can you change that to TS? Other than that, tnx for the PR

Reply:
since you included. `...` in the code. that is an invalid ts. So that I purposely mentioned bash there.

Reply:
But you did remove `. . .`. Didn't you? :)

Reply:
haha.😅 I didn't notice that

Reply:
I deleted the forked repo. You can merge and you can edit that yourself 😄

Reply:
done xD